tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -227,3 +238,47 @@ def test_loadbalancer_register_client_using_ak_to_ha_proxy(loadbalancer_setup, r
         {'organization-id': loadbalancer_setup['module_org'].id}
     )
     assert rhel7_contenthost.hostname in [host['name'] for host in hosts]
+
+
+@pytest.mark.rhel_ver_match('[^6]')
+@pytest.mark.tier1
+def test_client_register_through_lb(
+    loadbalancer_setup,
+    rhel_contenthost,
+    module_target_sat,
+    module_org,
+    module_ak,
+    module_location,
+):
+    """Register the client through loadbalancer
+
+    :id: c7e47d61-167b-4fc2-8d1a-d9a64350fdc4
+
+    :Steps:
+        1. Setup capsules, host and loadbalancer.
+        2. Generate curl command for host registration.
+        3. Register host through loadbalancer using global registration
+
+    :expectedresults: Global Registration should have option to register through
+    loadbalancer and host should get registered successfully.
+
+    :CaseLevel: Integration
+
+    :BZ: 1963266
+
+    :customerscenario: true
+    """
+    result = rhel_contenthost.register(
+        module_org,
+        module_location,
+        loadbalancer_setup['content_for_client']['client_ak'].name,
+        satellite=module_target_sat,
+        force=True,
+    )
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
+    assert module_target_sat.hostname in rhel_contenthost.subscription_config['server']['hostname']
```

Comment:
shouldn't we assert if its registered to load balancer?
```suggestion
    assert loadbalancer_setup['content_for_client']['client_ak'].name in rhel_contenthost.subscription_config['server']['hostname']
```